lib/codegen/CreateTerm.cpp

```diff
@@ -488,28 +499,48 @@ bool makeFunction(std::string name, KOREPattern *pattern, KOREDefinition *defini
     llvm::StringMap<llvm::Type *> params;
     std::vector<llvm::Type *> paramTypes;
     std::vector<std::string> paramNames;
+    std::vector<bool> needsLoad;
```

Comment:
This seems to be changing (or adding) a policy for how we call functions taking builtin collections,
passing a pointer directly to the off-heap collection value.
Is how we pass around collections documented somewhere?

Reply:
Not yet, but I don't want to do it now as I had to change it in the following pull request, so this code is actually deprecated.